docs/src/prototypes/popups/emojiPopup.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import {
+  Button,
+  Popup,
+  Grid,
+  Input,
+  popupFocusTrapBehavior,
+  gridBehavior,
+} from '@stardust-ui/react'
+import { emojiImages, renderImages } from './helper'
+import * as _ from 'lodash'
+
+const EmojiPopup = () => (
+  <Popup
+    accessibility={popupFocusTrapBehavior}
+    position="below"
+    trigger={<Button icon="smile" aria-label="Choose an emoji." />}
+    content={{
+      content: (
+        <div
+          aria-label="Choose an emoji. Press Enter key to insert emoji."
+          role="dialog"
+          aria-modal="true"
```

Comment:
let's see if we can create a Dialog (?) component and behavior that describes it, sets the role and aria-modal. The component can live within the prototypes/popup directory